src/DonationForms/resources/app/form/MultiStepForm/store/index.tsx

```diff
@@ -11,6 +11,7 @@ StoreContextDispatch.displayName = 'DonationFormMultiStepStateDispatch';
 export type MultiStepFormContextState = {
     steps: StepObject[];
     currentStep: number;
+    showHeader: true;
```

Comment:
<img width="1288" alt="Screenshot 2023-08-31 at 3 14 41 PM" src="https://github.com/impress-org/givewp/assets/10138447/9f227c24-903b-4a99-98da-285c39b894a2">


```suggestion
    showHeader: boolean;
```

Reply:
Resolved in b870e65a4